fred/__init__.py

```diff
@@ -129,7 +129,7 @@ async def on_error(self, event, *args, **kwargs):
             tbs = tbs + string
         tbs = tbs + "```"
         logging.error(tbs.replace("```", ""))
-        await self.get_channel(748229790825185311).send(tbs)
+        await self.get_channel(int(os.getenv("ERROR_CHANNEL"))).send(tbs)
```

Comment:
Please have the env var default to the previous value, I don't want to have to ask Vilsol to add the env var :P